src/NHibernate.Test/TestContainerSetup.cs

```diff
@@ -0,0 +1,76 @@
+namespace NHibernate.Test
+{
+	using System;
+	using System.Threading.Tasks;
+	using DotNet.Testcontainers.Containers;
+	using NUnit.Framework;
+	using Testcontainers.Db2;
+	using Testcontainers.FirebirdSql;
+	using Testcontainers.MariaDb;
+	using Testcontainers.MsSql;
+	using Testcontainers.MySql;
+	using Testcontainers.Oracle;
+	using Testcontainers.PostgreSql;
+
+	[SetUpFixture]
+	public class TestContainerSetup
+	{
+		private static IDatabaseContainer _container;
```

Comment:
When using double check locking, better set the tested variable as "volatile". Depending on the target architecture, it may be needed for preventing threading issues.

```suggestion
		private static volatile IDatabaseContainer _container;
```